database/migrations/2024_08_24_213634_create_courses_and_course_modules_tables.php

```diff
@@ -3,8 +3,9 @@
 use Illuminate\Database\Migrations\Migration;
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Support\Facades\Schema;
+use App\Domains\Course\Models\Course;
 
-class CreateCourseTable extends Migration
+class CreateCoursesAndCourseModulesTables extends Migration
```

Comment:
Usually, the good practice is not to modify the existing migration files. (See, the date is on the file name)

If you need any further changes (Ex: Change of column name, add or remove ant index, foreign key), it is recommended to create a new file.

Remind me to explain why it is important to have it in the next meeting

Reply:
I will create a separate migration file for course_modules.